app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,3 @@
+class RecipeIngredientsController < ApplicationController
```

Comment:
Good job creating this controller, but I would say try to not spend time on anything that's not needed _until you need it_. That way you make sure your focus is on TDD and fixing each error one at a time.